urbanopt/geojson-modelica-translator
# Add sphinx configuration for generating documentation

#### What does this PR accomplish?
Sets up sphinx documentation generator to document the GMT tool and adds a script for updating the documentation on github.
#### How should this be manually tested?
First, make sure the site looks decent. It should approximate the other urban opt geojson documentation sites as best as possible, e.g. [this one](https://docs.urbanopt.net/urbanopt-geojson-gem/rdoc/URBANopt/GeoJSON/Building.html).

Checkout the github pages site (to be added from the `gh-pages` branch) and verify all required elements are documented. Also try updating the documentation by editing a module/class/function docstring, then running `./docs/publish_docs.sh`, and verify the `gh-pages` branch was updated.
#### What are the relevant tickets?
#130 
#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2020-09-29 at 2 04 32 PM" src="https://user-images.githubusercontent.com/18518728/94609935-be6d1a00-025c-11eb-80ee-aa9476335fe1.png">
